src/overlays/actors/ovl_Obj_Tokei_Turret/z_obj_tokei_turret.c

```diff
@@ -28,23 +28,67 @@ const ActorInit Obj_Tokei_Turret_InitVars = {
     (ActorFunc)ObjTokeiTurret_Draw,
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_80B91EC0[] = {
+static InitChainEntry sInitChain[] = {
     ICHAIN_F32(uncullZoneForward, 1200, ICHAIN_CONTINUE),
     ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_STOP),
 };
 
-#endif
+void ObjTokeiTurret_Init(Actor* thisx, GlobalContext* globalCtx) {
+    ObjTokeiTurret* this = THIS;
+    Actor* actor;
+    s32 actorParams;
```

Comment:
Just `params` is fine, although looking through the rest, you ought to be able to identify the various values and make an enum easily enough.

Reply:
i have to admit, I'm not sure what this param actually resolves to.